docs/samples/Microsoft.ML.Samples/Dynamic/NgramExtraction.cs

```diff
@@ -49,21 +59,28 @@ public static void Example()
                     Console.WriteLine("");
                 }
 
-                Console.WriteLine("===================================================");
+                Console.WriteLine(
+                    "===================================================");
             };
             // Preview of the CharsUnigrams column obtained after processing the input.
             VBuffer<ReadOnlyMemory<char>> slotNames = default;
-            transformedData_onechars.Schema["CharsUnigrams"].GetSlotNames(ref slotNames);
-            var charsOneGramColumn = transformedData_onechars.GetColumn<VBuffer<float>>(transformedData_onechars.Schema["CharsUnigrams"]);
+            transformedData_onechars.Schema["CharsUnigrams"]
+                .GetSlotNames(ref slotNames);
```

Comment:
ugy ... change names to CamelCase. Maybe shorten name to avoid this ugliness

Reply:
hey @ganik the only thing this change does is wraps C# code at 85 character mark. There are many things we can improve with regards aesthetics but it isn't within the scope. There are many typos too in the samples code...

Reply:
sure, but going to public with such ugliness is not good. need to put best effort possible before deadline

---
In reply to: [278268213](https://github.com/dotnet/machinelearning/pull/3561#discussion_r278268213) [](ancestors = 278268213)

Reply:
Yes, we are doing best effort before the deadline. @shmoradims had specifically asked to only fix the line width. Lets see what he feels about this ugliness...